ci-operator/config/openstack-k8s-operators/openstack-operator/openstack-k8s-operators-openstack-operator-master.yaml

```diff
@@ -42,6 +46,14 @@ tests:
   commands: make gofmt || true
   container:
     from: src
+- as: precommit-check
+  commands: |
+    export GOFLAGS="-mod=mod"
+    export GOWORK=off
+    export GOLANGCI_LINT_CACHE=/tmp/.cache
+    pre-commit run --all-files --show-diff-on-failure
```

Comment:
With a simple `GOFLAGS=` pre-commit works for me without any go.work issues. Do I miss something?
```
gibi@bedrock:~/upstream/git/openstack-k8s-operators/openstack-operator D[tmp]                                                                                                                       [000] [15:19:41]
$ podman run -it --volume "`pwd`:/tmp/local" 36eb00cbe9e
[root@39f69867e672 origin]# cd /tmp/local
[root@39f69867e672 local]# GOFLAGS= pre-commit run --all-files --show-diff-on-failure
[INFO] Initializing environment for https://github.com/dnephin/pre-commit-golang.
[INFO] Initializing environment for https://github.com/pre-commit/pre-commit-hooks.
[INFO] Installing environment for https://github.com/pre-commit/pre-commit-hooks.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
make-manifests...........................................................Passed
make-generate............................................................Passed
make-operator-lint.......................................................Passed
go fmt...................................................................Passed
go vet...................................................................Passed
go-mod-tidy..............................................................Passed
go lint..................................................................Passed
check for added large files..............................................Passed
fix utf-8 byte order marker..............................................Passed
check for case conflicts.................................................Passed
check that executables have shebangs.....................................Passed
check that scripts with shebangs are executable..........................Passed
check for merge conflicts................................................Passed
check for broken symlinks............................(no files to check)Skipped
detect destroyed symlinks................................................Passed
check yaml...............................................................Passed
check json...............................................................Passed
detect private key.......................................................Passed
fix end of files.........................................................Passed
don't commit to branch...................................................Passed
trim trailing whitespace.................................................Passed
[root@39f69867e672 local]# 
```

Reply:
@gibizer it works for me locally on a branch as well. The issue is it is on master and the CI container is executed with a different user than it is local (probably locally running as root). Also CI is using 'master' for this /pj-rehearse job I think